### ISSUE TITLE
Loss curve

### ISSUE THREAD

**Issue by Dominic789654** [ShenDezhou/Open-GPT#1]
Hi,
Thanks for your excellent work!
Could you please share the training loss curve?

**Comment by ShenDezhou** [comment]
Hi Dominic, I think I've commit an image to show the loss.
